common/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -43,28 +43,21 @@ resource "template_dir" "resources" {
   }
 }
 
-resource "null_resource" "apply_stackdriver_monitoring" {
-  depends_on = ["template_dir.resources"]
-
-  triggers = {
-    nonce = "${var.nonce}"
-  }
-
+resource "null_resource" "destroy_stackdriver_lbm" {
   provisioner "local-exec" {
     command = <<EOF
       export PROJECT_ID=${var.project_id}
       export GOOGLE_CLOUD_KEYFILE=${var.serviceaccount_key}
-      export STACKDRIVER_DEBUG=${var.stackdriver_debug}
 
-      RETRIES=10
+      RETRIES=5
       RETRY_COUNT=1
       while [ "$STACKDRIVER_DEADLINE_EXCEEDED" != "false" ]; do
         STACKDRIVER_DEADLINE_EXCEEDED="false"
-        echo "[Try $RETRY_COUNT of $RETRIES] Applying Stackdriver resources..."
+        echo "[Try $RETRY_COUNT of $RETRIES] Destroying Stackdriver resources..."
         bundle exec ruby -e '
           require "/rakefiles/stackdriver.rb"
-          resources = read_resources("${path.module}/resources_rendered")
-          apply_resources(resources)
+          destroy_resources({"alert_policies"=>[],"notification_channels"=>[]})
+          destroy_uptime_checks(["${join("\",\"", google_monitoring_uptime_check_config.this.*.name)}"])
```

Comment:
I think code might fail, as there's no `google_monitoring_uptime_check_config.this` resource in this module.